content/docs/reactivesearch/react-searchbox/searchbox.md

```diff
@@ -0,0 +1,262 @@
+    searched value might appear in multiple fields of the same document, this is true only if you have configured multiple fields in `dataField` prop. Defaults to `true`.
+    <br/> <br/>
+    **Example** if you have `showDistinctSuggestions` is set to `false` and have following configurations
+
+        ```js
+        // Your document:
+        {
+            "name": "Warn",
+            "address": "Washington"
+        }
+        // Component:
+        <Searchbox dataField=['name', 'address'] ... />
+        // Search Query:
+        "wa"
+        ```
+
+    Then there will be 2 suggestions from the same document
+    as we have the search term present in both the fields
+    specified in `dataField`.
+
+    ```
+    Warn
+    Washington
+    ```
+
+#### To configure the SearchBox API
+
+-   **id** `string` [required] a unique identifier for the query can be referenced in the react property of other queries.
+
+-   **dataField** `dataFieldValidator`
+    database field(s) to be queried against. Accepts a String or an Array of either String or `DataField` type. The latter is useful for searching across multiple fields with field weights.<br/>
+    Think of field weights as a way to apply weighted search. To use field weights, you can define the `dataField` prop as an array of objects of `DataField` type.<br/>
+    The `DataField` type has the following shape:
+
+    > Note:
+    > This prop is optional only when `enableAppbase` prop is set to `true`.
+
+-   **aggregationField** `string`
+    One of the most important use-cases this enables is showing `DISTINCT` results (useful when you are dealing with sessions, events and logs type data).
+    It utilizes `composite aggregations` which are newly introduced in ES v6 and offer vast performance benefits over a traditional terms aggregation.
+    You can read more about it over [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-composite-aggregation.html).
+    You can use `aggregationData` using `onAggregationData` callback.
+
+    ```javascript
+    <SearchBox
+        app="good-book-ds-latest"
+        credentials="IPM14ICqp:8e573e86-8802-4a27-a7a1-4c7d0c62c186"
+        dataField="original_title"
+        aggregationField="original_title.keyword"
+        onAggregationData={(next, prev) => <>}
+    />
+    ```
+
+    > See impact of aggregationField with these example for [React](/docs/reactivesearch/v3/advanced/groupingresults/#how).
+
+-   **nestedField** `string`
+    Set the path of the `nested` type under which the `dataField` is present. Only applicable only when the field(s) specified in the `dataField` is(are) present under a [`nested` type](https://www.elastic.co/guide/en/elasticsearch/reference/current/nested.html) mapping.
+
+-   **size** `number` To set the number of results to be returned by a query.
```

Comment:
@nikhil-varma There are formatting differences b/w the prop whitespaces (description is on next line in Searchbase API ref v/s same line over here)